### PR TITLE
fix(cli): handle redirections from dev portal

### PR DESCRIPTION
**Pull request by tanguyantoine**
## 🐻 What your PR is doing?

support login redirection

## 📦 Package concerned
- @bearer/cli

## 🛠 How to test it

- link and run bearer login

## ✅ Checklist

- [ ] Tests were added (if necessary)
- [x] I used conventional commits

### CODE REVIEW

File: packages/cli/src/commands/login.ts [hunk 1]
```diff
@@ -108,31 +109,28 @@ export default class Login extends BaseCommand {
...
-              this.stopServer()
-              this.getToken(data.code)
-            } catch (e) {
-              this.debug(e)
-            }
-          })
-        })
-        .listen(BEARER_LOGIN_PORT, () => resolve(server))
+      const app = express()
+      app.use((_req: express.Request, res: express.Response, next: express.NextFunction) => {
```
Comment (RadekMolenda) [inline packages/cli/src/commands/login.ts]:
```
import { Request, Response, NextFunction } from 'express'
```
then
```suggestion
      app.use((_req: Request, res: Response, next: NextFunction) => {
```

File: packages/cli/src/commands/login.ts [hunk 2]
```diff
@@ -108,31 +109,28 @@ export default class Login extends BaseCommand {
...
+        res.setHeader('Connection', 'close')
+        next()
+      })
+      app.get('/login/callback', (req: express.Request, res: express.Response) => {
```
Comment (RadekMolenda) [inline packages/cli/src/commands/login.ts]:
```suggestion
      app.get('/login/callback', (req: Request, res: Response) => {
```

File: packages/cli/src/commands/setup/auth.ts [hunk 3]
```diff
@@ -153,14 +153,21 @@ export default class SetupAuth extends BaseCommand {
       }
       this.debug('starting server')
       const app = express()
-      app.get('/setup_auth_callback', (req: express.Request, res: express.Response) => {
+      app.use((_req: express.Request, res: express.Response, next: express.NextFunction) => {
```
Comment (RadekMolenda) [inline packages/cli/src/commands/setup/auth.ts]:
```suggestion
      app.use((_req: Request, res: Response, next: NextFunction) => {
```

File: packages/cli/src/commands/setup/auth.ts [hunk 4]
```diff
@@ -153,14 +153,21 @@ export default class SetupAuth extends BaseCommand {
...
+        res.setHeader('Connection', 'close')
+        next()
+      })
+      app.get('/setup/auth-callback', (req: express.Request, res: express.Response) => {
```
Comment (RadekMolenda) [inline packages/cli/src/commands/setup/auth.ts]:
```suggestion
      app.get('/setup/auth-callback', (req: Request, res: Response) => {
```

**Comment by RadekMolenda** [comment]
@tanguyantoine please fix the build first

File: packages/cli/src/commands/login.ts (hunk 1 above)
Comment (tanguyantoine) [reply packages/cli/src/commands/login.ts]: I can't because esModuleInterop isn't enabled

**Pull request merged**